runelite-client/src/main/java/net/runelite/client/plugins/worldhopper/WorldSwitcherPanel.java

```diff
@@ -63,6 +64,8 @@
 	private ArrayList<WorldTableRow> rows = new ArrayList<>();
 	private WorldHopperPlugin plugin;
 
+	private Map<World, Integer> pingHistory = new HashMap<>();

```

Comment:
I'm not familiar with all the datatypes in java. Is this the best implementation of what I'm trying to achieve here?

Reply:
Probably